src/2048-expert/index.html

```diff
@@ -13,6 +13,31 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+            The aim of the game is to get 2048
+            <br>
+            <br>
+            Use the arrow keys to move the tiles
+            <br>
+            Restart a new game by pressing enter
+            <br>
+            Tiles with the same value will get merged after moves
+            <br>
+            <br>When a tile has the value <strong>2048</strong>
+            <br> Congradulations! <strong>You Win!</strong>
+            <br>
+            <br>Start a new game and challenge your limits!
+        </p>
+        <div class="modal-header">Expert version</div>
```

Comment:
`Expert version` as a separate header might imply that there is another game mode that you can switch to so I think that a single header is enough.